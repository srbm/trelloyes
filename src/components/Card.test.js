import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card test suite', () => {
  it(' Card renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card key={1} title="Test Title" content={[1,2,3]} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders Card UI as expected', () => {
    const tree = renderer
      .create(<Card key={1} title="Test Title" content={[1,2,3]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
