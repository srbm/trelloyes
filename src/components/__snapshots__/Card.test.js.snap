// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card test suite renders Card UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Test Title
  </h3>
  <p>
    1
    2
    3
  </p>
</div>
`;
